javascript/javascript1/week1/js1homework.js

```diff
@@ -0,0 +1,119 @@
+//Excercise1
+const yearOfBirth = 1980;
+const futureYear = 2025;
+const userName = "Reenu";
+const currentAge = new Date().getFullYear() - yearOfBirth;
+const futureAge = futureYear - yearOfBirth;
+
+console.log(userName + " is " + currentAge + " years old now.");
+console.log("She will be " + futureAge + " years old in " + futureYear);
+
+//Excercise2
+//"1 dog year = 7 human years"
+
+const dogYearOfBirth = 2015;
+const dogYearFuture = 2025;
+let shouldShowResultInDogYears = true;
+const dogAgeInFuture = dogYearFuture - dogYearOfBirth;
+if (shouldShowResultInDogYears) {
+  console.log(
+    `Your dog will be ${dogAgeInFuture * 7} dog years old in ${dogYearFuture}`
+  );
+} else {
+  console.log(
+    `Your dog will be ${dogAgeInFuture} human years old in ${dogYearFuture}`
+  );
+}
+
+//Excercise 3
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const houseDimensionPeter = [8, 10, 10];
+const houseVolumePeter =
+  houseDimensionPeter[0] * houseDimensionPeter[1] * houseDimensionPeter[2];
+const gardenSizePeter = 100;
+const calculatedHousePricePeter =
+  houseVolumePeter * 2.5 * 1000 + gardenSizePeter * 300;
+const actualPricePaidPeter = 2500000;
+
+console.log("Calculated volume for Peter's house " + houseVolumePeter);
+console.log("Calculated price for Peter's house " + calculatedHousePricePeter);
+console.log("Actual price paid for Peter's house " + actualPricePaidPeter);
+
+if (calculatedHousePricePeter > actualPricePaidPeter) {
+  console.log(
+    `Peter paid 
+      ${calculatedHousePricePeter - actualPricePaidPeter}
+       less than the calculated price. `
+  );
+} else if (calculatedHousePricePeter == actualPricePaidPeter) {
+  console.log(
+    `Peter paid ${actualPricePaidPeter}. Same as the calculated price`
+  );
+} else {
+  console.log(
+    `Peter paid   
```

Comment:
There is a strange format of the messages, I am not sure if it was attentional or not.